src/main/java/com/thealgorithms/others/ReverseArray.java

```diff
@@ -0,0 +1,45 @@
+// Created by Pronay Debnath
```

Comment:
Why this is in the same PR?

Reply:
O that should be in different PR, sorry

Reply:
I removed the file from this PR, could you now check the two files? @debasishbsws